### DISCUSSION TITLE
ERROR is occuring

### DISCUSSION THREAD

**Discussion by hrdagr** [Thomasbehan/LesNet#37] [Bugs]
![Scre2enshot 2024-04-21 224724](https://github.com/Thomasbehan/SkinVestigatorAI/assets/86142689/0cba47d1-28c1-4d31-ad6a-f8dadd0c9181)

the project is going to an error even in the live demo page

 raise TypeError(
TypeError: Unexpected keyword: reason='You cannot set the body to a text value without a charset'

**Comment by Thomasbehan** [comment]
Thanks for letting me know, I'll get this fixed asap 🙌

**Comment by Thomasbehan** [comment]
Thanks @hrdagr, a fix is now being merged into the main branch and will be deployed live shortly after
![image](https://github.com/Thomasbehan/SkinVestigatorAI/assets/8860273/2a24bf8a-ac5d-429f-94ac-3f910558512f)
